markdown/blog/neuroglia_project_interview.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "AsyncAPI Made Simple: Neuroglia’s Innovative Tools for Developers"
+date: 2025-04-28T06:00:00+01:00
+type: Communication
+tags:
+  - Project Status
+  - Interview
+  - Communinty
+cover: 
+authors:
+  - name: Atinuke Oluwabamikemi Kayode
+    photo: /img/avatars/bami.webp
+    link: https://www.linkedin.com/in/atinuke-oluwabamikemi-kayode-5b838b1b7/
+    byline: AsyncAPI Community Marketing Specialist
+excerpt: 'A comprehensive review of AsyncAPI Marketing Working Group achievements in 2024, including social media growth, store closure updates, and future initiatives like the Student Ambassador Program'
+---
+
+We recently had the pleasure of interviewing Charles d'Avernas, who co-founded Neuroglia SRL alongside Jean-Baptiste Bianchi, a company known for providing elegant and straightforward solutions to complex IT challenges. Charles and his team work with prominent players in the IT community, such as Cisco, to create sophisticated event-driven solutions that address core business issues.
```

Comment:
[suggestion]: I think it’d be a good idea to hyperlink Neuroglia and the names of its founders, just in case readers want to learn more about them.